server/src/routes/series.ts

```diff
@@ -0,0 +1,9 @@
+import * as express from "express";
+import seriesController from "@controllers/seriesController";
+
+const seriesRouter = express.Router();
+
+seriesRouter.get("/getall", seriesController.getAllSeries);
```

Comment:
```suggestion
seriesRouter.get("/", seriesController.getAllSeries);
```

It's not according to the REST